src/Servers.cpp

```diff
@@ -1826,7 +1826,16 @@ void Server::rpc_blockchain_transaction_broadcast(Client *c, const RPC::BatchId
     // -- filter may end up applying murmur3 hash to the entire key we give it from 20-50 times!
     const QByteArray txkey = (!rawtxhex.isEmpty() ? BTC::HashOnce(Util::ParseHexFast(rawtxhex)).left(16) : QByteArrayLiteral("xx"));
     // no need to validate hex here -- bitcoind does validation for us!
-    generic_async_to_bitcoind(c, batchId, m.id, "sendrawtransaction", QVariantList{ rawtxhex },
+    QVariantList params { rawtxhex } ;
+    if ((bitcoindmgr->isCoreLike())
```

Comment:
No need for those  parens.

```suggestion
    if (bitcoindmgr->isCoreLike()
```